From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Adopt newer package dependencies.

Since Debian has moved to newer versions of Flask, Jinja2 and Werkzeug,
this package also needs to adopt these newer dependencies, with Moin
itself needing upstream work to migrate to them.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 78edb5d..b25f31c 100644
--- a/setup.py
+++ b/setup.py
@@ -79,7 +79,7 @@ setup_args = dict(
         'emeraldtree>=0.10.0',  # xml processing
         'feedgen==0.9.*',  # Atom feed
         'flatland>=0.8',  # form handling
-        'Jinja2<3.1.0',  # template engine
+        'Jinja2>=3.1.0',  # template engine
         'markupsafe<=2.2.0',  # safe html and xml
         'pygments>=1.4',  # src code / text file highlighting
         'Werkzeug<2.4.0',  # wsgi toolkit
